app/Http/Controllers/Auth/LoginController.php

```diff
@@ -63,6 +63,14 @@ protected function verifyLogin(Request $request) {
         ) {
             return redirect('/')->withError("We need you to grant us all marked permissions");
         }
+        if(session('realops_redirect')) {
```

Comment:
Do we want to add a guard here to make sure the toggle is enabled? Just to prevent some kind of backdoor login with it disabled (I'm not sure if this would even matter, but just a thought)

Reply:
Yeah, that's a solid idea. I'll do that now